### DISCUSSION TITLE
Stepper driver 5v output to power controller

### DISCUSSION THREAD

**Discussion by lowv** [kachurovskiy/nanoels#104] [Q&A]
Hi,
I'm currently building the H2 version with a CL57t closed loop driver and was wondering if it's possible to use its 5V encoder output to power the nanoELS?Couldn't find any info on the max load capacity of the output pin, so maybe it's best to use a seperate 48v to 5v step down converter? 

Kind regards

**Comment by kachurovskiy** [comment]
Hi @lowv, I have no problem using the ELS when powered through Arduino USB port. On/Off switch doesn't work in that case but everything else is functional. Cheers!
